lib/client-assets.php

```diff
@@ -652,7 +652,7 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	$gutenberg_theme_support = get_theme_support( 'gutenberg' );
 	$color_palette = gutenberg_color_palette();
 
-	if ( $gutenberg_theme_support && $gutenberg_theme_support[0]['colors'] ) {
+	if ( $gutenberg_theme_support && isset( $gutenberg_theme_support[0]['colors'] ) && $gutenberg_theme_support[0]['colors'] ) {
```

Comment:
If we're going to test having been set and "truthy"-ness, would it be more concise to use `! empty` instead?

```php
if ( $gutenberg_theme_support && ! empty( $gutenberg_theme_support[0]['colors'] ) ) {
```

http://php.net/manual/en/types.comparisons.php